src/python_framework_v02/troute/nhd_io.py

```diff
@@ -904,13 +905,19 @@ def write_channel_restart_to_wrf_hydro(
     -------
     """
     # create t-route simulation timestamp array
-    with xr.open_dataset(channel_initial_states_file) as ds:
-        t0 = ds.Restart_Time.replace("_", " ")
+    # TODO: Replace this with a more general function to identify
```

Comment:
- [ ] Make issue to capture this TODO

Reply:
This is now reading a t0 from the input, which is pulled from a general function.